### PR TITLE
Use electron-winstaller

### PR DESCRIPTION
**Pull request by paulcbetts**
This PR moves us to use electron-winstaller instead of duplicating the code between the Grunt task and the library version

### CODE REVIEW

**Comment by paulcbetts** [comment]
I think this PR is correct but it looks like on Windows we're hitting a weirdo bug in electron-winstaller

File: package.json [hunk 1]
```diff
@@ -13,9 +13,10 @@
     "test": "grunt test"
   },
   "dependencies": {
+    "asar": "~0.8.0",
```
Comment (kevinsawicki) [inline package.json]: I think the asar and underscore dependencies can be completely removed now.

**Comment by kevinsawicki** [comment]
Went ahead and got this off CoffeeScript and a grunt-based build as well in this pull request, thanks for getting this going.

**Comment by havenchyk** [comment]
Hey @kevinsawicki! Is there anything else that should be done in this PR? I polished PR a little bit in the #107 on top of the current one.

**Comment by paulcbetts** [comment]
@kevinsawicki Once this build is :gem: this should finally be good to go, sorry this took forever

**Comment by kevinsawicki** [comment]
:+1:

**Comment by kevinsawicki** [comment]
Published as `2.0.0`

**Pull request merged**